### PR TITLE
Avatar

### PR DESCRIPTION
**Pull request by Veekshith-Thoram**
image

### CODE REVIEW

File: homepage/main.css [hunk 1]
```diff
@@ -0,0 +1,16 @@
+:root{
+    --primary-color: #34D399;
+    --secondary-color: ;
+}
+
+body{
+    /* background-color: var(--primary-color); */
```
Comment (AtulPant2704) [inline homepage/main.css]: Remove this comment.

File: homepage/main.css [hunk 2]
```diff
@@ -0,0 +1,16 @@
+:root{
+    --primary-color: #34D399;
+    --secondary-color: ;
```
Comment (AtulPant2704) [inline homepage/main.css]: You have not assigned any color to this variable, I think this will result in an error.

File: avatar/avatar.css [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+    padding: 0rem 0.5rem;
+}
+
+.heading-text{
+    font-size: 1.3rem;
+    font-weight: bold;
+}
+.links{
+    text-decoration: none;
+    /* margin-top: 3rem; */
```
Comment (AtulPant2704) [inline avatar/avatar.css]: Remove this comment as well.

File: avatar/avatar.css [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+    font-weight: bold;
+    font-size: 2rem;
+    
+}
+
+.nav-pills{
+    text-align: right;
+}
+.list-non-bullet{
+    list-style: none;
```
Comment (AtulPant2704) [inline avatar/avatar.css]: This you could have done in your reset.css file itself, you could have given `ul` `list-style: none` over there and then imported that file in all the files where you wanted to use that, you don't need to write an explicit class just for that.

**Review by AtulPant2704 — CHANGES_REQUESTED** [review]
- You have included `alert.html` in this branch, from next time onwards try to have files specific to that branch in that PR.
- Remove all the unnecessary comments.
- Assign a color to the `secondary-color` or remove that line itself if it is of no use.
- From next PR, only have screenshot of the UI page itself, not of the code. We are gonna review the code anyway.

File: avatar/avatar.css [hunk 5]
```diff
@@ -0,0 +1,151 @@
+@import url('https://fonts.googleapis.com/css2?family=Fredoka&display=swap');
+
+:root{
+    --primary-color: #34D399;
+    --secondary-color: ;
+    
+}
+
+
```
Comment (AnkurChunekar) [inline avatar/avatar.css]: Please try to be consistent in line spacing and indentation.

File: avatar/avatar.html [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+                    <ul class="avatar-ul"><img src="/avatar/images/spud icon.jpg" id="avatar-md" alt="avatar"></ul>
+                    <ul class="avatar-ul"><img src="/avatar/images/spud icon.jpg" id="avatar-sm" alt="avatar"></ul>
+                </li>
+
+                <div class="desc-txt">
+                    By using the below code snippet you can create avatar with image on it of your choice just by
+                    changing the src in the img tag.
+                </div>
+                <iframe src="https://www.thiscodeworks.com/embed/622efdececc7d70015fadfe2"
+                    style="width: 70%; height: 125px;" frameborder="0"></iframe>
```
Comment (AnkurChunekar) [inline avatar/avatar.html]: Avoid inline styling wherever possible.

File: avatar/avatar.html [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+
+                <div class="line"></div>
+
+                <h2>Avatar with Text</h2>
+
+
+                <div class="text-container">
```
Comment (AnkurChunekar) [inline avatar/avatar.html:77]: Here also indentation and spacing is inconsistent, use a code formatter. I use prettier that might help.

**Review by AnkurChunekar — CHANGES_REQUESTED** [review]

**Review by NagaSatyaGeeth — APPROVED** [review]

**Pull request merged**